### PR TITLE
Updated arrow function to get string from server

### PR DESCRIPTION
**Pull request by kolbsara**
Updated website to fetch greeting from server using arrow function, part 2 complete

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -21,6 +21,10 @@ <h1>Sara's Portfolio</h1>
       <div>
         <p>Click <a href="/data">here</a> to view content generated by a servlet.</p>
       </div>
+      <div>
+        <button onclick="getGreetingUsingArrow()">Get greeting!</button>
```
Comment (sbahaud) [inline portfolio/src/main/webapp/index.html:25]: I would prefer just calling this getGreeting. The fact that it uses the Arrow function is something that is internal to the function and there is no value in exposing it.

File: walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/index.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```
Comment (sbahaud) [inline walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/index.html:13]: I didn't see this function in the js file. 
Also name the function just getRandomQuote.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (kolbsara) [reply portfolio/src/main/webapp/index.html:25]: Done

File: walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/index.html (hunk 2 above)
Comment (kolbsara) [reply walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/index.html:13]: getRandomQuoteUsingArrowFunctions was part of the starter code, we were supposed to swap out the functions as practice using arrow functions.

Comment (sbahaud) [reply walkthroughs/week-2-server/examples/random-quotes/src/main/webapp/index.html:13]: you can remove the function if it is no longer used. Ok to leave it for now.

**Pull request merged**